bitgouel-api/src/main/kotlin/team/msg/domain/government/presentation/request/CreateGovernmentRequestData.kt

```diff
@@ -0,0 +1,8 @@
+package team.msg.domain.government.presentation.request
+
+import team.msg.common.enums.Field
+
+data class CreateGovernmentRequestData (
```

Comment:
```suggestion
data class CreateGovernmentRequestData(
```

```
ClassName(
```

이 컨벤션이 잘 맞춰지지 않은 곳이 몇 부분 있습니다! 확인하고 수정 부탁드립니다 ~

Reply:
[9b0f4fe](https://github.com/School-of-Company/Bitgouel-Server/pull/483/commits/9b0f4fec88f4bfb5a74036ca0efd688bcca5effc) 컨벤션 수정했어요